src/components/Game.vue

```diff
@@ -718,8 +765,13 @@ export default {
             const middle = this.archivesMode ? `archive${wordID > 0 ? ` #${wordID}`:''} [${this.archivesDate.format('DD/MM/YYYY')}]` : `#${wordID}`
 
             const title = `Le Mot (@WordleFR) ${middle} ${this.currentAttempt <= NB_ATTEMPTS ? this.currentAttempt : '💀' }/${NB_ATTEMPTS}\n\n`;
+            let count = 0;
             let schema = this.results.slice(0, this.currentAttempt).map((result) => {
+                if (this.womenRightsDay) count++;
                 return result.map((letter) => {
+                    if (count === this.currentAttempt && letter === 'correct') {
+                        return '🟪';
+                    }
```

Comment:
Purement une question de pdv et d'intelligibilité du code, mais je te laisse décider de l'appliquer ou non :)
```suggestion
            let schema = this.results.slice(0, this.currentAttempt).map((result, index) => {
                return result.map((letter) => {
                    if (this.womenRightsDay && (index + 1) === this.currentAttempt && letter === 'correct') {
                        return '🟪';
                    }
```